docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:16-alpine
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
```

Comment:
The volume mapping for `my_db:$PGDATA` assumes that `$PGDATA` is set correctly in the environment. Ensure that this environment variable is defined in your `.env` file or elsewhere in your configuration.